tests/conftest.py

```diff
@@ -0,0 +1,40 @@
+import glob
+import json
+import os
+
+import pytest
+from nomad.utils import structlogging
+
+
+@pytest.fixture(scope='function')
+def capture_from_logger(caplog):
+    """
+    Extracts log messages from the logger and raises an assertion error if any
+    ERROR messages are found.
+    """
+    caplog.handler.formatter = structlogging.ConsoleFormatter()
+    yield caplog
+    for record in caplog.get_records(when='call'):
+        if record.levelname in ['ERROR']:
+            try:
+                msg = structlogging.ConsoleFormatter.serialize(json.loads(record.msg))
+            except Exception:
+                msg = record.msg
+            assert False, msg
+
+
+@pytest.fixture(scope='function')
+def clean_up():
```

Comment:
Can you ask @ladinesa or @JFRudzinski how the handle this in their testings? I remember that there were some heaving MD testing in Gitlab which generated extra files if ran locally, but not during the pipeline.

Reply:
Ah, nice! Having this discussion would be helpful indeed!